src/pages/video-listing/video-listing.js

```diff
@@ -70,19 +119,46 @@ export const VideoListing=()=>{
             <div className="showing ">
                 All videos
             </div>
+
+                {modal?<div className="modal">
+                <div className="modal-header p-1">
+                <h3 className="">Save to playlist</h3>
+                <div className='playlist-parent' onClick={()=>setModal(false)}><span class="material-icons">cancel</span></div>
+                 </div>
+                <hr />
+                <div className="modal-body">
+                    {playlist.map(item=>(
+                        <div key={item._id}>
+                            <input onChange={()=>onChangeHandler(item._id)} type="checkbox" />
+                            <span>{item.title}</span>
+                        </div>
+                    ))}
+                </div>
+                {playlistName?<div>
+                    <label htmlFor="">Name</label>
+                    <input onChange={(e)=>setPlaylistInput(e.target.value)} type="text" placeholder='enter playlist name' />
+                </div>:""}
+                <hr />
```

Comment:
&& can be used instead of ? to simplify the code.

Reply:
yes right,will do it.